presentation/tests.py

```diff
@@ -1,3 +1,44 @@
+from django.core.urlresolvers import reverse
+from django.test import Client
 from django.test import TestCase
 
-# Create your tests here.
+from presentation.models import Presentation
+from warp.users.models import User
+
+
+class PresentationListTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.test_user = self.create_test_user()
+        self.presentation = Presentation(subject="subject",
+                                         author=self.test_user,
+                                         markdown="#Abcdefghijklmno",
+                                         html="<h1>Abcdefghijklmno</h1>")
+        for _ in range(0, 20):
+            self.presentation.save()
+
+    def test_get_presentation_list_page(self):
+        response = self.client.get(reverse('presentation:list'))
+
+        self.assertContains(response, self.presentation.subject)
```

Comment:
I don't agree using random value, but I agree @mingrammer 's opinion that `subject="subject"` is not a proper test data. 
How about change the test data more 'real-like'. Imagine some use case on production. For example,
```python
self.presentation = Presentation(subject="I love you, mother!", 
    author=self.test_user, 
    markdown=~~~~~~blabla~~~~~~, 
    html=~~~~blabla~~~~)
```
Of course, It makes test code verbose. But there is solution on Django. [The fixture](https://docs.djangoproject.com/en/1.10/topics/testing/tools/#fixture-loading).